ext/CMakeLists.txt

```diff
@@ -3,7 +3,11 @@ project(genesis-ext)
 find_package(Threads)
 
 if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/src")
-    message(STATUS "SRC directory already extracted! Skipping...")
+    # delete the src directory if it exists
+    message(STATUS "Deleting SRC directory!")
+    file(REMOVE_RECURSE "${CMAKE_CURRENT_SOURCE_DIR}/src")
+    message(STATUS "Extracting SRC directory!")
```

Comment:
Just FYI, since these messages appear "flat" in the CMake log, it would seem that they are about the root `src` directory. Better to say "Extracting ext/src" or something.